ethereum_bridge/Cargo.toml

```
@@ -51,5 +51,7 @@ tracing = "0.1.30"
 
 [dev-dependencies]
 assert_matches = "1.5.0"
+data-encoding = "2.3.2"
+ethabi = "17.0.0"
```

shouldn't we be using 18.0.0?
```suggestion
ethabi = "18.0.0"
```